cmd/root.go

```diff
@@ -26,10 +17,20 @@ func Execute(chainsYaml []byte) {
 		panic(fmt.Errorf("error parsing chains.yaml: %v", err))
 	}
 
+	var rootCmd = &cobra.Command{
+		Use:   "heighliner",
+		Short: "Generate docker images for Cosmos chains",
+		Long: `Welcome to Heighliner, provided by Strangelove Ventures.
+
+This tool can generate docker images for all different release versions
+of the configured Cosmos blockchains in chains.yaml`,
+	}
+
+	rootCmd.AddCommand(BuildCmd())
+	rootCmd.AddCommand(ListCmd())
```

Comment:
Instead, it's super easy to initialize everything scoped to `main()`. That way, the execution flow (in this case registering subcommands) is in one spot. No indirection.